### PR TITLE
flake8 tests passing

### PR DESCRIPTION
**Pull request by czpython**

### CODE REVIEW

File: cmsplugin_filer_file/conf.py [hunk 1]
```diff
@@ -1,6 +1,5 @@
-#-*- coding: utf-8 -*-
-from django.conf import settings
-from django.utils.translation import ugettext_lazy as _
+# -*- coding: utf-8 -*-
+from django.conf import settings  # NOQA
```
Comment (mikek) [inline cmsplugin_filer_file/conf.py]: What was wrong with this line?

File: tox.ini [hunk 2]
```diff
@@ -25,6 +25,6 @@ commands = flake8
 basepython = python2.7
 
 [flake8]
-ignore = E251,E128
+ignore = E251,E128,E501
```
Comment (mikek) [inline tox.ini]: Nice, but can we increase the `max-line-length` to some reasonable value instead?

**Comment by mikek** [comment]
LGTM. 
Take a look at those ultra paranoid checks from QuantifiedCode. It might be worth disabling some of them.

File: tox.ini (hunk 2 above)
Comment (czpython) [inline tox.ini]: line length checks should have never been enabled.. so agreed with Martin to disable for now until we can fix them

**Pull request merged**